packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -39,12 +46,15 @@ import ItemActions from './item-actions';
 import { ENUMERATION_TYPE, OPERATOR_IN } from './constants';
 
 const {
-	DropdownMenuV2,
-	DropdownMenuGroupV2,
-	DropdownMenuItemV2,
-	DropdownMenuSeparatorV2,
-	DropdownSubMenuV2,
-	DropdownSubMenuTriggerV2,
+	CompositeV2: Composite,
+	CompositeRowV2: CompositeRow,
+	useCompositeStoreV2: useCompositeStore,
+	CompositeItemV2: CompositeItem,
+
+	DropdownMenuV2Ariakit: DropdownMenu,
+	DropdownMenuGroupV2Ariakit: DropdownMenuGroup,
+	DropdownMenuItemV2Ariakit: DropdownMenuItem,
+	DropdownMenuSeparatorV2Ariakit: DropdownMenuSeparator,
```

Comment:
Migrating to the new `Ariakit`-based dropdown menu, which plays better with `Composite`.